File: 프로그래머스/lv0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String numbs = my_string.toLowerCase().replaceAll("[a-z]", "");
        for (int i = 0; i < numbs.length(); i++) {
            answer += Integer.parseInt(numbs.substring(i, i + 1));
        }
        return answer;
    }
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.kt
class Solution {
    fun solution(my_string: String): Int {
        var answer: Int = 0
        for(i in 0 until my_string.length){
            if(my_string[i] in '1'..'9'){
                answer += my_string[i].toString().toInt()
            }
        }
        
        return answer
    }
}